config/config.yaml

```diff
@@ -106,14 +106,15 @@ Router:
   kafkaDialTimeout: 10s
   minRetryBackoff: 10s
   maxRetryBackoff: 300s
-  noOfWorkers: 64
+  noOfWorkers: 1
   allowAbortedUserJobsCountForProcessing: 1
   maxFailedCountForJob: 3
   retryTimeWindow: 180m
   failedKeysEnabled: true
   saveDestinationResponseOverride: false
   transformerProxy: false
   transformerProxyRetryCount: 15
+  oAuthV2Enabled: true
```

Comment:
why do we need that?

Reply:
This is some local changes got pushed. Removed those.